src/cli/cli.cc

```diff
@@ -2172,6 +2197,61 @@ int main (const int argc, const char* argv[]) {
       );
     }
 
+    if (settings.count("build_copy_map") != 0) {
+      auto copyMapFile = fs::path{settings["build_copy_map"]};
+      if (!fs::exists(fs::status(copyMapFile))) {
+        log("warning: file specified in [build] copy_map does not exist");
+      } else {
+        auto copyMap = trim(readFile(copyMapFile));
+        auto copyMapFileDirectory = fs::absolute(copyMapFile.relative_path().parent_path());
+
+        if (copyMap.size() > 0) {
+          auto lines = split(copyMap, '\n');
+          for (const auto& line : lines) {
+            // handle :|#|// leading comments
+            if (line.starts_with(":") || line.starts_with("#") || line.starts_with("//")) {
```

Comment:
Is there a reason for not adding this functionality to `parseConfig()`

If so, how about implementing this parser functionality in another function so we can use it for `.sscrc` as per discord?

Reply:
well it is INI without sections, more just `key = value`

Reply:
Even if you aren't using sections, wouldn't the two feature sets (`parseConfig` and what you have implemented) still be compatible and have a fair bit of overlap?
It looks as though `parseConfig` has support for empty prefixes (global section)
Also you didn't answer the query about at least implementing a separate function that we can use for a `.sscrc` file as well as this mapping config.

Reply:
what is `.sscrc`?